relay_messages_test.go

```diff
@@ -89,12 +192,6 @@ func TestLazyCallReqRejectsOtherFrames(t *testing.T) {
 	}, "Should panic when creating lazyCallReq from non-callReq frame.")
```

Comment:
nit: can we add a general `Panics` function for each of the lazy types, I want to make sure we don't miss the validation there.

This test can probably be simplified now to use the other types to generate a frame and verify that it panics on that frame, rather than using init req.